### PR TITLE
Updating fonts and sounds settings

### PR DESCRIPTION
**Pull request by gmontimantis**
No LP, DIG revamp project.

### CODE REVIEW

File: docs/modules/admin/pages/workstation_admin.adoc [hunk 1]
```diff
@@ -21,33 +21,22 @@ on the workstation.
...
-
-It is also possible to zoom in and zoom out when viewing the OPAC in the
-staff client, making the font appear larger or smaller. (This will not
+It is possible to zoom in and zoom out when viewing Evergreen on the staff
+side and OPAC, making the font appear larger or smaller. (This will not
 affect other screens.) Use *CTRL + +* (plus sign, to zoom in), *CTRL + -*
 (minus sign, to zoom out), and *CTRL + 0* (to restore default). The
 workstation will remember the setting.
 
+=== Disable Settings ===
```
Comment (sandbergja) [inline docs/modules/admin/pages/workstation_admin.adoc:32]:
```suggestion
=== Disable Sounds ===
```

Should this be disable sounds?

**Review by sandbergja — COMMENTED** [review]
Thanks, @gmontimantis !  It looked good to me, just one question about a heading.

File: docs/modules/admin/pages/workstation_admin.adoc (hunk 1 above)
Comment (gmontimantis) [reply docs/modules/admin/pages/workstation_admin.adoc]: Hi Jane!  Sorry for the long wait.  I pushed a correction.

**Comment by sandbergja** [comment]
Thanks, @gmontimantis !  Cherry-picked to rel_3_9 and above.

**Pull request closed without merging**